docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -136,109 +181,415 @@ These variables apply only when using NCO mode (i.e. when ``RUN_ENVIR`` is set t
+
+..
+   COMMENT: Needs review. 
+
+``SPP_STDDEV_CUTOFF``: (Default: ( "1.5" "1.5" "2.5" "1.5" "1.5" ) )
+   Perturbation magnitude cutoff in number of standard deviations from the mean. 
+
+..
+   COMMENT: Needs review. 
+
+``SPP_VAR_LIST``: (Default: ( "pbl" "sfc" "mp" "rad" "gwd" ) )
+   The list of parameterizations to perturb: planetary boundary layer (PBL), surface physics (SFC), microphysics (MP), radiation (RAD), gravity wave drag (GWD). Valid values: "pbl", "sfc", "rad", "gwd", and "mp".
+
+..
+   COMMENT: Needs review. Is "rad" radiation? Need confiromation. 
+
+
+Land Surface Model (LSM) SPP
+-------------------------------
+
+Turn on SPP in Noah or RUC LSM (support for Noah MP is in progress). Please be aware of the :term:`SDF` that you choose if you wish to turn on Land Surface Model (LSM) SPP. SPP in LSM schemes is handled in the ``&nam_sfcperts`` namelist block instead of in ``&nam_sppperts``, where all other SPP is implemented. The default perturbation frequency is determined by the ``fhcyc`` namelist entry. Since that parameter is set to zero in the SRW App, use ``LSM_SPP_EACH_STEP`` to perturb every time step. 
+
+LSM perturbations include soil moisture content [SMC] (volume fraction), vegetation fraction (VGF), albedo (ALB), salinity (SAL), emissivity (EMI), surface roughness (ZOL) (in cm), and soil temperature (STC). Perturbations to soil moisture content (SMC) are only applied at the first time step. Only five perturbations at a time can be applied currently, but all seven are shown below. In addition, only one unique *iseed* value is allowed at the moment, and it is used for each pattern.
+
+``DO_LSM_SPP``: (Default: "false") 
+   Turns on Land Surface Model (LSM) Stochastic Physics Parameterizations (SPP). When "TRUE", sets ``lndp_type=2``, which applies land perturbations to the selected paramaters using a newer scheme designed for data assimilation (DA) ensemble spread. LSM SPP perturbs uncertain land surface fields ("smc" "vgf" "alb" "sal" "emi" "zol" "stc") based on recommendations from physics experts. 
+
+``LSM_SPP_TSCALE``: (Default: ( ( "21600" "21600" "21600" "21600" "21600" "21600" "21600" ) )
+   Decorrelation timescale in seconds. 
+
+``LSM_SPP_LSCALE``: (Default: ( ( "150000" "150000" "150000" "150000" "150000" "150000" "150000" ) )
+   Decorrelation spatial scale in meters.
+
+``ISEED_LSM_SPP``: (Default: ("9") )
+   Seed to initialize the random perturbation pattern.
+
+``LSM_SPP_VAR_LIST``: (Default: ( ( "smc" "vgf" "alb" "sal" "emi" "zol" "stc" ) )
+   Indicates which LSM variables to perturb. 
+
+``LSM_SPP_MAG_LIST``: (Default: ( ( "0.2" "0.001" "0.001" "0.001" "0.001" "0.001" "0.2" ) )
+   Sets the maximum random pattern amplitude for each of the LSM perturbations. 
+
+``LSM_SPP_EACH_STEP``: (Default: "true") 
+   When set to "TRUE", it sets ``lndp_each_step=.true.`` and perturbs each time step. 
```

Comment:
@JeffBeck-NOAA @ClaraDraper-NOAA With the LSM PR going in (1) will the LSM_SPP_EACH_STEP parameter be renamed in the SRW App? (2) is it still a binary true/false variable (looks like there are options 0-3 for lndp_model_type) (3) if lndp_each_step is being replaced by lndp_model_type, what happens when LSM_SPP_EACH_STEP (or its equivalent) is set to true? Here's my best guess: 
``LSM_SPP_EACH_STEP``: (Default: "true") 
   When set to "TRUE", it sets ``lndp_model_type=2`` and perturbs each time step.

Reply:
@JeffBeck-NOAA My PR should go through next week I think. There are still 2 in the queue in front of me, I think.